### ISSUE TITLE
Suppression Fields

### ISSUE THREAD

**Issue by SW50000** [BEEmod/BEE2-items#2009]
This is an item that goes on and off, and when it's active, it will kill the player if inside.
![20170526162252_1](https://cloud.githubusercontent.com/assets/20195894/26513228/06ffcdc6-4230-11e7-96c4-eb4faa7f7523.jpg)
(I know packing custom particles is impossible, but I was wondering if there is an alternative)

**Comment by ZombiePwner1** [comment]
@SW50000 I'm afraid theres no alternative.

**Comment by Luke18033** [comment]
@ZombiePwner1 No, there are. For one thing, we could just make it a texture, but that wouldn't look as good. We could also make it a model, which uses a custom effect sprite textured onto a bunch of moving rectangles. If you ever check Discord you'll see that @TeamSpen210 isn't even sure particles are unpackable, in which case we just use that.

**Comment by LautaroL20** [comment]
@Luke18033 - No, he never checks Discord. I think u maked him admin for nothing lol.

**Comment by SW50000** [comment]
> I think u maked him admin for nothing lol.

*Shots Fired*

**Comment by SW50000** [comment]
Also, here's something Luke said on discord:

> We'd probably use a texture, or a model using a custom sprite like TeamSpen210 suggested

**Comment by ZombiePwner1** [comment]
@LautaroL20 I do check discord i just leave it closed most of the time.

**Comment by Byzarru** [comment]
Isn't this just a death field? Or does it make the entire area impassable?

**Comment by Konclan** [comment]
@Byzarru Its on a timer, it flashes periodically, its basically a crusher except brighter

**Comment by TeamSpen210** [comment]
So a laserfield with timer? I'd prefer to implement a timer item, and hook that up directly .

**Comment by SW50000** [comment]
I prefer an item that takes up the entire block for it. I don't like the look of a cluster of laserfeilds.

**Comment by Endermage77** [comment]
So it's a laser field variant of a crusher. Noice.
Actually, it would be cool to have area versions of crushers too that can place like cutout tiles.

**Comment by hop193** [comment]
What mod is that?

**Comment by Luke18033** [comment]
Portal Prelude.

**Comment by IceTudor** [comment]
So simply just a SA electric pad but with yellow particles and glow.

**Comment by Luke18033** [comment]
We can now pack particles, though these would probably be replaced by the Flash Version/Still Alive zappers.
